tutorials/large/L2_large_link_prediction.py

```diff
@@ -218,8 +218,8 @@ def forward(self, g, h):
 
 
 ######################################################################
-# Evaluating Performance (Optional)
-# ---------------------------------
```

Comment:
I feel this sentence needs improvement. ", where it treats the node embeddings learned by link prediction via training and evaluating a linear classifier on top of the learned node embeddings."

Reply:
Rephrased it.